subprojects/docs/src/samples/userguide/dependencyManagement/artifactTransforms/relocate/kotlin/build.gradle.kts

```diff
@@ -50,7 +50,7 @@ abstract class ClassRelocator : TransformAction<ClassRelocator.Parameters> {
     override
     fun transform(outputs: TransformOutputs) {
         val primaryInputFile = primaryInput.get().asFile
-        if (parameters.externalClasspath.contains(primaryInput)) {           // <6>
+        if (parameters.externalClasspath.contains<Any>(primaryInput)) {      // <6>
```

Comment:
```
Script compilation error:

  Line 53:         if (parameters.externalClasspath.contains(primaryInput)) {            
                                                    ^ Type inference failed. The value of the type parameter T should be mentioned in input types (argument types, receiver type or expected type). Try to specify it explicitly.

1 error
```

This is the inference engine being more strict. `contains()` comes from Kotlin stdlib `Iterable<T>.contains(T)`, its signature didn't change between 1.3.31 and 1.3.40.

This can be reproduced with 
```kotlin
listOf("42").contains(23)
```

Reply:
It seems to be a legit error. The code should read `.contains(primaryInputFile)`, no?